coordinator/core/src/main/kotlin/net/consensys/zkevm/ethereum/coordination/aggregation/GlobalAggregationCalculator.kt

```diff
@@ -7,8 +7,8 @@ import net.consensys.zkevm.domain.BlobCounters
 import net.consensys.zkevm.domain.BlobsToAggregate
 import org.apache.logging.log4j.LogManager
 import org.apache.logging.log4j.Logger
-import java.lang.IllegalStateException
 import java.util.PriorityQueue
+import java.util.concurrent.atomic.AtomicInteger
 
 class GlobalAggregationCalculator(
```

Comment:
Would it be easier to capture these metrics in `ProofAggregationCoordinatorService::onAggregation` instead?

Reply:
Yes, indeed, given that aggregation interval in `onAggregation` is final.